addons/default/visiosoft/advs-module/resources/views/new-ad/preview/preview.twig

```diff
@@ -0,0 +1,41 @@
+{% extends "theme::layouts/default" %}
+
+{% block content %}
+    <div class="pt-5">
+        <p class="preview-paragraph">
+            {{ trans('visiosoft.module.advs::field.preview_text.part_1') }}
```

Comment:
part_1 is meaingless what is it? @Diatrex  Give a name

Reply:
the name is preview_text, it has two sentences, I want to put a break line tag between them, there is no meaningful name for them, I can name them sentence 1 and sentences 2

Reply:
ok then